rxp-hpp-android/build.gradle

```diff
@@ -19,9 +19,9 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.google.code.gson:gson:2.5'
-    compile 'com.android.support:appcompat-v7:23.1.1'
-    compile 'com.squareup.retrofit:retrofit:1.7.1'
-    compile 'org.apache.httpcomponents:httpcore:4.4.4'
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.google.code.gson:gson:2.5'
+    implementation 'androidx.appcompat:appcompat:1.0.0'

```

Comment:
To confirm, is `androidx.appcompat` a drop-in replacement for `com.android.support`?

Reply:
Yes. AndroidX is part of JetPack , its a new thing for maintaining support libraries for android native.